packages/api/tests/endpoints/test-replays.js

```diff
@@ -115,15 +115,7 @@ test.serial('request to replays endpoint with valid kinesis parameters starts an
   t.true(asyncOperationStartStub.calledOnce);
   t.is(lambdaName, process.env.ManualConsumerLambda);
   t.is(cluster, process.env.EcsCluster);
-  t.deepEqual(payload, {
-    CollectionsTable: process.env.CollectionsTable,
-    RulesTable: process.env.RulesTable,
-    ProvidersTable: process.env.ProvidersTable,
-    stackName: process.env.stackName,
-    system_bucket: process.env.system_bucket,
-    FallbackTopicArn: process.env.KinesisFallbackTopicArn,
-    ...body
-  });
+  t.deepEqual(payload, body);
```

Comment:
Did this not work...?

Reply:
It did, but I needed to add the `defaultSchedulerQueueUrl` environment variable into the async operation. And I noticed that someone added this `useLambdaEnvironmentVariables` flag which automatically copies all the environment variables from the Lambda into the async operation ECS task. That seemed preferable to me then continuing to pass all the environment variables through from the Lambda to the ECS task by hand.

See the corresponding changes to `endpoints/replays.js` and `lambda/manual-consumer.js`

As a result of all of those changes, this test had to be updated